core/kubernetes/README.md

```diff
@@ -29,3 +29,14 @@ Once you are in the correct working directory (`core/kubernetes/`) we can procee
 ./setup.sh
 ```
 
+To run in development mode, where the OpenHIM mongo database, HAPI fhir server and the MySQL database can be accessed directly through their urls, run the following command
+
+```bash
+./setup.sh dev
```

Comment:
I dont think we should have the `dev` flag as part of the `main/setup.sh` script. This script should be in its own `dev` folder. We could add a question in this bash script to ask if they have started the `main` scripts so that this can be executed on top of the existing services